Create new file with schedule for 2019-2020

mchiapello opened the pull request:
As Tobin suggested, I created a new file with 2019-2020 schedule

sstevens2 commented:
Hi @mchiapello!
Sorry but I'm confused about this PR.  It looks like there are two files changed one for 2018-2019 and one for 2019-2020 but they both have overlapping dates in 2019 which don't seem to match?

Also all the lines for events are missing a closing `|`.

mchiapello commented:
Hi @sstevens2 ,
the file for 2018-2019 contains the correct dates for 2018 and proposed dates for 2019. The files 2019-2020 contains the real dates for 2019 and proposed dates for 2020. The two files do not match because the real dates for 2019 have been delayed. Initially, I changed the 2018-2019 file, but Tobin Magle suggested me to create a new file for 2019-2020. 

I added the missing `|`.

sstevens2 commented:
Thanks for adding the `|`'s.  

I'm still concerned about leaving the proposed dates up if they aren't true anymore.  Seems like someone could find those and be confused.  And we can always use the history if we want to look back at the old pages.

mchiapello commented:
Hi @sstevens2,
I removed the 2019 dates from 2018-2019-schedule.md file. Now the dates for 2019 are only in one place, the 2019-2020-schedule.md file.

sstevens2 commented:
I think that looks great but don’t have write access to this repo, so I can’t merge it.  @kariljordan ?

kariljordan commented:
I made a team for you all so that I can give you write access. [Here is the team](https://github.com/orgs/carpentries/teams/instructor-development-committee). For now I'll merge this. Please accept the team invite!

sstevens2 commented:
Thanks @Kariljordan!!
